oauth2/src/test/groovy/com/okta/spring/boot/oauth/AutoConfigConditionalTest.groovy

```diff
@@ -265,7 +261,8 @@ class AutoConfigConditionalTest implements HttpMock {
         webContextRunner().withPropertyValues(
                 "okta.oauth2.issuer=https://test.example.com/",
                 "spring.security.oauth2.client.provider.okta.issuerUri=${mockBaseUrl()}", // work around to not validate the https url
-                "okta.oauth2.client-id=test-client-id")
+                "okta.oauth2.client-id=test-client-id",
+                "okta.oauth2.client-secret=test-client-secret")
```

Comment:
Why was the secret added here?

Reply:
Because the issuer used here is root org. Therefore, Opaque flow is chosen and this requires `client-secret` to be set otherwise the test will fail.

Reply:
update the test property`okta.oauth2.issuer` to be something that the previous code was expected (so the test functions the same way), then add a new test (if needed for the newer code)

This will ensure the previous functionality continues to work unchanged

Reply:
Also, this is a PKCE test, that requires that no secret be present
(currently, this is a limitation with Spring Security)

Reply:
I'm attempting to push a fix for this